services/IoT/applications/api.md

```diff
@@ -29,8 +29,8 @@ The {{site.data.keyword.iot_short_notm}} API supports the following functions fo
 - Device diagnostic operations (clear logs, retrieve logs, add log information, delete logs, get specific logs, clear error codes, get device error codes, and add error codes)
```

Comment:
line 17
"Use the {{site.data.keyword.iot_full}} API to build and customize applications that interact 

with your organization on {{site.data.keyword.iot_short_notm}}."

Should it be the HTTP API? That seems more specific. It seems like any API could be called the 

Watson IoT Platform API.


line 25 (the list)
ideally, all list items should start with the same part of speech